openmetadata-ui/src/main/resources/ui/src/utils/GlobalSettingsUtils.tsx

```diff
@@ -40,17 +40,15 @@ export const getGlobalSettingMenus = (
     icon,
     children: children
       ? children
-          .filter(({ isProtected }) => {
-            return hasAccess ?? !isProtected;
-          })
-          .map(({ label, icon }) =>
-            getGlobalSettingMenus(
+          .filter((menu) => (hasAccess ? menu : !menu.isProtected))
+          .map(({ label, icon }) => {
```

Comment:
If possible please avoid negative conditions.

Reply:
I don't think so we can avoid it in this case as we need to send a false for filtering out